KubeArmor/core/kubeArmor.go

```diff
@@ -449,6 +449,11 @@ func KubeArmor() {
 
 	// == //
 
+	if !dm.K8sEnabled && cfg.GlobalCfg.Policy {
```

Comment:
Since we are using `cfg.GlobalCfg.Policy` I don't think there's any need for ContainerPolicy. 
No k8s and enabling KubeArmorPolicy should be suffice to indiciate it is for Containers right? WDYT?

Reply:
Yes, that would be simpler.